pandas/core/frame.py

```diff
@@ -674,9 +675,11 @@ def _repr_html_(self):
             max_rows = get_option("display.max_rows")
             max_cols = get_option("display.max_columns")
             show_dimensions = get_option("display.show_dimensions")
+            unique_id = str(uuid1())
 
             return self.to_html(max_rows=max_rows, max_cols=max_cols,
-                                show_dimensions=show_dimensions, notebook=True)
+                                show_dimensions=show_dimensions, notebook=True,
+                                table_id=unique_id)
```

Comment:
What's the benefit of having a random table_id for tables in the notebook? I'd prefer to not have them unless there's a compelling reason.

Reply:
My thought was that it can't hurt since the id is almost certain not to conflict with another and  it might make it easier for people to apply custom styles/javascript to their notebook. Because the id is random, however, I can see how this might be hard and I will get rid of the feature (for now) (more pull requests to come :) ). Updated in the most recent pull request